interface/src/app/gene/network/network.js

```diff
@@ -149,31 +151,18 @@ angular.module('adage.gene.network', [
         }
       };
 
-      // After machine learning model in the URL has been validated,
-      // reconfigure the slider and force render it. See the discussion at:
-      // https://github.com/angular-slider/angularjs-slider/issues/79#issuecomment-225438841
-      // (Also tried $scope.$$postDigest, not work.)
-      var refreshSlider = function() {
-        $timeout(function() {
-          $scope.$broadcast('rzSliderForceRender');
-        }, 1000);
-        // Note by dhu: 1000ms is an arbitrary timeout value. The minimum
-        // timeout value that worked on my desktop is 150ms.
-      };
-      // Monitor "self.isValidModel". Once it is true, reconfigure a few slider
-      // parameters and call refreshSlider().
-      $scope.$watch(
-        function() {
-          return self.isValidModel;
-        },
-        function() {
-          if (self.isValidModel) {
-            self.slider.min = MlModelTracker.g2gEdgeCutoff;
-            self.slider.options.floor = MlModelTracker.g2gEdgeCutoff;
-            refreshSlider();
+      self.maxGeneNumSlider = {  // slider that controls max number of genes
+        value: 0,                // initial position of slider
+        options: {
+          floor: 1,              // minimum of the slider bar
+          ceil: 100,             // maximum of the slider bar
```

Comment:
What are the units on these numbers? We are not limiting to 100 genes max, right? Is this a percentage?

Reply:
"1" and "100" are just plain numbers without unit. `100` is kind of like a placeholder here. On line 254, it is reset to the actual number of genes in the network.